Makefile

```diff
@@ -3,6 +3,7 @@
 .PHONY: test
 test:
 	./gradlew clean assemble check --stacktrace
+	./gradlew compileJmhJava
```

Comment:
I would create a different Makefile target for the benchmarks. This way, we can decouple benchmarks from compilation in CircleCI

Reply:
agree